### PR TITLE
Make map setup formatting for coop scenarios consistent with official books

### PR DESCRIPTION
**Pull request by Dima793**
Resolves qwrtln/Homm3BG-mission-book#145 
Changes "Map setup" section formatting for coop scenarios from "bold text with bullet points" to "just bullet points", to be consistent with how it's done in Archon Studio's official HoMM books.

### CODE REVIEW

**Comment by qwrtln** [comment]
Thanks for the PR, here's the diff:

en-06
en-09
en-13

**Comment by qwrtln** [comment]
Yeah, I think it works 👍🏻 
Let me fix the runners first, as it seems the CI stopped working after GitHub rolled out Ubuntu 24.10.

File: coops/sentinels.tex [hunk 1]
```diff
@@ -40,17 +40,10 @@ \subsection*{\MakeUppercase{Map Setup}}
...
-\end{itemize}
-
-\textbf{4 × Far (II--III) Map Tile}
-
-\textbf{4 × Near (IV--V) Map Tile}
-\begin{itemize}
-  \item All Near Map Tiles must have Obelisks
+	\item 2 × Starting (I) Map Tile
+	\item 4 × Far (II--III) Map Tile
+	\item 4 × Near (IV--V) Map Tile with the Obelisk field
```
Comment (qwrtln) [inline coops/sentinels.tex:46]:
```suggestion
	\item 4 × Near (IV--V) Map Tile with the Obelisk Field
```

**Comment by qwrtln** [comment]
Huh, it seems the runners don't work on PRs from forks. Some GitHub mumbo-jumbo, that is.

**Comment by qwrtln** [comment]
Mind reopening this as a collaborator? I'll send you the invitation.

**Comment by Dima793** [comment]
> Sentinels changes cascade

I wasn't sure on your policy for artificial column/page breaks, so didn't insert any, sorry.

**Comment by Dima793** [comment]
> Mind reopening this as a collaborator? I'll send you the invitation.

Do I have to open it in any specific way? Or just perform the same actions while having a different status this time?

**Comment by qwrtln** [comment]
> I wasn't sure on your policy for artificial column/page breaks, so didn't insert any, sorry.

The policy is common sense, no worries here.

**Comment by qwrtln** [comment]
> Do I have to open it in any specific way? Or just perform the same actions while having a different status this time? UPD: I guess I should create a branch in your repository, with no connection to any forks. But is there anything else to change?

Just reopen, then we'll look at the working CI jobs and carry on from there.

**Pull request closed without merging**